internal/ansible/handler/logging_enqueue_owner_test.go

```diff
@@ -1,269 +1,258 @@
-// Copyright 2021 The Operator-SDK Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/*
+	Copyright 2018 The Kubernetes Authors.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+			http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
 
 package handler
 
 import (
+	"context"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllertest"
 	"sigs.k8s.io/controller-runtime/pkg/event"
-	crHandler "sigs.k8s.io/controller-runtime/pkg/handler"
-
-	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-var _ = Describe("LoggingEnqueueRequestForOwner", func() {
+var _ = Describe("Eventhandler", func() {
+	var ctx = context.Background()
 	var q workqueue.RateLimitingInterface
-	var instance LoggingEnqueueRequestForOwner
 	var pod *corev1.Pod
-	var podOwner *metav1.OwnerReference
-
+	var mapper meta.RESTMapper
 	BeforeEach(func() {
-		q = controllertest.Queue{Interface: workqueue.New()}
-		podOwner = &metav1.OwnerReference{
-			Kind:       "Pod",
-			APIVersion: "v1",
-			Name:       "podOwnerName",
-		}
-
+		q = &controllertest.Queue{Interface: workqueue.New()}
 		pod = &corev1.Pod{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace:       "biz",
-				Name:            "biz",
-				OwnerReferences: []metav1.OwnerReference{*podOwner},
-			},
+			ObjectMeta: metav1.ObjectMeta{Namespace: "biz", Name: "baz"},
 		}
-
 		pod.SetGroupVersionKind(schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"})
+		Expect(cfg).NotTo(BeNil())
```

Comment:
Actually this needs envtest setup to be able to get the config from it. Are we doing it anywhere in this pkg? (I maybe missing it)

Reply:
Like this: https://github.com/kubernetes-sigs/kubebuilder/blob/ce24b89a23fb8a51be78b115c15fd0f08975b46a/testdata/project-v4/internal/controller/suite_test.go#L71

Reply:
[51bf8c6](https://github.com/operator-framework/ansible-operator-plugins/pull/37/commits/51bf8c65da97216b34f19330cd72833790c5fee5)